bigtop-deploy/puppet/manifests/bigtop_repo.pp

```diff
@@ -43,9 +43,10 @@
       # I couldn't enforce the sequence -> anymore because of this
       # https://twitter.com/c0sin/status/875869783979196416
        apt::conf { "disable_keys":
-          content => "APT::Get::AllowUnauthenticated 1;",
+          content => "APT::Get::AllowUnauthenticated 1;\nAcquire::AllowInsecureRepositories \"true\";",
```

Comment:
Is there a reason why we have these two? Asking as a genuine question, I am new to BigTop but I wouldn't have expected something like this..

Reply:
@elukey I did some trace and it was originally added in:
https://issues.apache.org/jira/browse/BIGTOP-1676
I'm the author. I guess that's for testing purpose. Anyhow it seems not secure as what you're concerning. We should make it default to secure mode and disable if needed in test.

Reply:
Alright. I'm working on the fix. BIGTOP-3355 is to track this. How about we leave it as-is here and fix it in BIGTOP-3355 @sekikn @elukey.

Reply:
+1 thanks a lot!

Reply:
Thanks for the comment @elukey and the fix @evans-ye, I'll rebase this PR after #639 is merged.